planning/behavior_path_planner_common/src/turn_signal_decider.cpp

```diff
@@ -667,16 +702,19 @@ std::pair<TurnSignalInfo, bool> TurnSignalDecider::getBehaviorTurnSignalInfo(
 
   const auto relative_shift_length = end_shift_length - start_shift_length;
 
+  const auto p_path_start = getPose(path.path.points.front());
+  const auto p_path_end = getPose(path.path.points.back());
+
   // If shift length is shorter than the threshold, it does not need to turn on blinkers
   if (std::fabs(relative_shift_length) < p.turn_signal_shift_length_threshold) {
-    return std::make_pair(TurnSignalInfo{}, true);
+    return std::make_pair(TurnSignalInfo(getPose(p_path_start), getPose(p_path_end)), true);
```

Comment:
Ota-san has already called  `getPose` from the path points and assigned them to `p_path_start` and `p_path_end`.
Is there any reason why you called `getPose` again  on these two variables?

Reply:
:tashikani:

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/7045/commits/1959d3fd77b0a1f2503c8aa3a7eece23dd57f4e9